internal/middleware/logger.go

```diff
@@ -31,6 +31,8 @@ func LoggerMiddleware(rootLogger *zerolog.Logger) func(next http.Handler) http.H
 				Logger()
 			t1 := time.Now()
 
+			logger.Debug().Msg("Started request")
```

Comment:
```suggestion
			logger.Debug().Msgf("Started %s request %s", r.Method, r.URL.Path)
```

Reply:
What for? we have these in the logger itself :thinking:

Reply:
see my comment ^^ :)

Reply:
I pasted an example screenshot above, see how useful this is. I would argue that we already have this for "Completed" message, for consistency let's please keep it the same way. I find it very useful myself.

In general, I agree this should not be the way we create logs, but this middleware is special - all requests have these logs.

Reply:
oki, you're right it doesn't hurt :)